[23기 배윤아] Modify : 회원가입 페이지 레이아웃 수정 2

WinterYoon opened the pull request:
>wecode PR message template 및 체크 리스트 입니다. 
아래 사항들을 전부 작성/체크 하시고 PR 해주세요!

## 수정 사항 간략한 한줄 요약
8/3 회원가입 페이지 레이아웃 작성 완료

8/4 회원가입 페이지 레이아웃 수정
8/4 3시 34분 추가 회원가입 페이지 레이아웃 재수정
		 
## 수정 사항들 자세한 내용
8/3 HTML & CSS 작성 완료
8/3 백엔드와 소통중 유효성 검사 로직이 정확히 정해지지 않아, 정해지면 레이아웃 내 약간의 수정이 있을 예정입니다. 

8/4 테이블 태그 사용한 회원가입 폼 양식 수정, float 속성 flex로 대체
8/4 3시 34분 array.map 과 상수 데이터로 리팩토링

## 기타 질문 및 특이 사항

8/4 수정 후 git add -> commiit -> push 하는 과정에 push를 진행했으나 github에 pr이 뜨지않고 진행이 되지않아서 git pull origin feature/signupLayout 으로 pull을 받은 후 컨플릭트 발생 -> 해결 후 git push 했는데 Modify 내역이 두개가 올라갔는데 괜찮을까요?  --> 확인했습니다 감사합니다 


## 체크 리스트 (아래 사항들이 전부 체크되어야만 merge가 됩니다!)
- [x] 필요한 test들을 완료하였고 기능이 제대로 실행되는지 확인 하였습니다.
- [x] Wecode의 코드 스타일 가이드에 맞추어 코드를 작성 하였습니다.
- [x] 제가 의도한 파일들과 수정 사항들만 커밋이 된 것을 확인 하였습니다.
- [x] 본 수정 사항들을 팀원들과 사전에 상의하였고 팀원들 모두 해당 PR에 대하여 알고 있습니다.

Commit:
Add : 회원가입 페이지 레이아웃 작성 완료

Commit:
Add : 회원가입 페이지 레이아웃 작성 완료

yeonuk-hwang commented on src/pages/Signup/Signup.js:
```diff
@@ -0,0 +1,116 @@
+        </div>
+        <div className="signUpContainer">
+          <form>
+            <table className="signUpTable" summary="회원 기본정보">
```
- Login form 안에 table 형태로 들어가야 하는 이유는 없는 것 같습니다!
- 회원가입 페이지에 table로 표현할 부분은 아닌 것 같습니다!

WinterYoon replied:
제가 잘 이해하지 못했는데 , table 태그로 감싼 부분(tr, th 전부) 을 아예 지워야되는건가요? 지금 form > table > tr, th... 등으로 묶인 상태여서 아예 테이블 형식을 지우고 div 박스로 새로 레이아웃을 짜야되는 것인지 궁금합니다..!

WinterYoon replied:
테이블 태그 없애고 수정했습니다!

yeonuk-hwang commented on src/pages/Signup/Signup.js:
```diff
@@ -0,0 +1,116 @@
+class Signup extends React.Component {
+  render() {
+    return (
+      <div className="signUp">
```
전체를 다 div로 감싸는 것 보다는, main등이 좋지 않을까요? 이 페이지에서 제일 핵심적인 부분인 것 같네요!

WinterYoon replied:
div 없애고 main으로 감쌌습니다!

yeonuk-hwang commented on src/pages/Signup/Signup.scss:
```diff
@@ -1 +1,89 @@
\ No newline at end of file
+.signUp {
+  margin-top: 100px;
+  width: 2000px;
```
width를 2000px로 특정하신 이유가 있나요?

WinterYoon replied:
아 큰 모니터로 width 꽉 채워서 보려다가 임시로 입력한게 그대로 올라간 것 같습니다! 100%로 수정했습니다!

yeonuk-hwang commented on src/pages/Signup/Signup.scss:
```diff
@@ -1 +1,89 @@
+
+    .signUpEssential {
+      display: inline-block;
+      float: right;
```
float 속성은 유지보수가 힘들어서 과거에만 쓰이던 속성입니다. CSS3에서는 float을 쓸 일이 없습니다! 다른 속성으로 대체해주세요!

WinterYoon replied:
display: flex 활용해서 정렬 했습니다!

yeonuk-hwang commented on src/pages/Signup/Signup.js:
```diff
@@ -0,0 +1,116 @@
+                  <td>
+                    <input type="password" className="signUpPwInput" />
+                    <span>
+                      {' '}
```
불필요한 부분 같습니다!

WinterYoon replied:
띄어쓰기로 margin을 주려고 했는데 저렇게 올라갔습니다.  직접 값 추가했습니다!

yeonuk-hwang commented on src/pages/Signup/Signup.js:
```diff
@@ -0,0 +1,116 @@
+                </tr>
+                <tr>
+                  <th scope="row">
+                    이메일 수신여부{' '}
```
{' '} 이것도 불필요한 것 같습니다.

WinterYoon replied:
위와 동일합니다!!!!

yeonuk-hwang reviewed (commented):
브랜치 잘 나눠서 작업하시는 것 👍
리뷰 확인 및 반영해주세요~

Commit:
Modify: 회원가입 페이지 레이아웃 수정

Commit:
Modify : 회원가입 페이지 레이아웃 수정

Merge branch 'feature/signupLayout' of https://github.com/wecode-bootcamp-korea/23-1st-Weeteweete-frontend into feature/signupLayout

yeonuk-hwang commented on src/pages/Signup/Signup.js:
```diff
@@ -0,0 +1,92 @@
+        </div>
+        <div className="signUpContainer">
+          <form className="signUpForm">
+            <div className="signUpFormId">
+              <span className="signUpFormIndex">
+                아이디<span className="signUpEssentialMark">*</span>
+              </span>
+              <input type="text" />
+              <span>(영문소문자/숫자, 4~16자)</span>
+            </div>
+            <div className="signUpFormPw">
+              <span className="signUpFormIndex">
+                비밀번호<span className="signUpEssentialMark">*</span>
+              </span>
+              <input type="password" />
+              <span>
+                (영문 대소문자/숫자/특수문자 중 2가지 이상 조합, 10자~16자)
+              </span>
+            </div>
```
전체적으로 인풋창 여러개가 반복되는 모습으로 보이네요!
상수데이터 + Array.map method 활용해서 한번 표현해보세요!

WinterYoon replied:
수정했습니다!

yeonuk-hwang commented on src/pages/Signup/Signup.scss:
```diff
@@ -1 +1,90 @@
\ No newline at end of file
+.signUp {
+  margin-top: 100px;
+  width: 100%;
```
width:100% 는 signUp이 이미 display:block의 속성을 가지고 있기에 불필요해보입니다!
그리고 max-width:100vw도 명시적으로 값을 100vw를 넘어서는 값을 주지않는 이상 불필요해보입니다(이 전에 width:2000px 이 있을 경우에는 필요했겠지만, 지금은 불필요해보이네요)
화면을 꽉 채우는 걸 원하신다면 그냥 width:100vw로 설정하시면 될 것 같습니다!

WinterYoon replied:
수정했습니다!!!

yeonuk-hwang reviewed (commented):
피드백 반영 잘 해주시고 코멘트도 남겨주셔서 너무 좋네요👍
table tag 없어지니까 훨씬 깔끔해졌고요!
리뷰 확인하시고 반영해주세요

8/4 수정 후 git add -> commiit -> push 하는 과정에 push를 진행했으나 github에 pr이 뜨지않고 진행이 되지않아서 git pull origin feature/signupLayout 으로 pull을 받은 후 컨플릭트 발생 -> 해결 후 git push 했는데 Modify 내역이 두개가 올라갔는데 괜찮을까요?
-> push 가 왜 안됬는지 의문이네요?... 아마 중간에 뭔가 잘못됬던 것 같습니다 그래도 잘 해결해주셨네요! 현재 컨플릭트는 안나고 있어서 괜찮아보이고, 커밋 2개쌓인건, 커밋 하나로 합치는 기능도 있긴 한데 지금은 그냥 넘어가셔도 될 것 같습니다!

Commit:
Modify : 회원가입 레이아웃 수정 2

yeonuk-hwang reviewed (approved):
LGTM

yeonuk-hwang commented:
저 너무 미워하진 마세요 : ) ㅋㅋㅋㅋ
PR 제목은 매번 수정하실 필요 없습니다, 제일 마지막 수정사항을 PR 제목에 적는 것이아니라, 이 브랜치의 작업내용을 통합해서 한줄로 정리해주시면 됩니다.
세부 작업내용들은 PR 본문에 기입해주시면 되요~
